rubygem-algebrick/rubygem-algebrick.spec

```diff
@@ -15,13 +15,15 @@ Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 Requires: %{?scl_prefix}ruby(rubygems)
 Requires: %{?scl_prefix}ruby
 Requires: %{?scl_prefix}rubygem(activesupport)
```

Comment:
This dependency needs to be removed, seems it went a while ago and we missed it.